KPI/dashboards/desktop_kpis_with_filters_alternate.dashboard.lookml

```diff
@@ -450,8 +449,13 @@
     series_types: {}
     title_hidden: true
     listen:
+      Channel: firefox_desktop_usage_2021.channel
+      Activity Segment: firefox_desktop_usage_2021.activity_segment
+      OS: firefox_desktop_usage_2021.os
+      Attributed (Yes / No): firefox_desktop_usage_2021.attributed
+      Country Name: firefox_desktop_usage_2021.country_name
```

Comment:
This fixes the issue Klukas noticed, where topline numbers weren't updated when filters changed